packages/stores/types.ts

```diff
@@ -285,9 +305,8 @@ export type TCoreStores = {
     common: TCommonStore;
     menu: TMenuStore;
     ui: TUiStore;
-    // This should be `any` as this property will be handled in each package.
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    modules: any;
+    modules: Record<string, any>;
+    contract_store: TContractStore;
```

Comment:
@niloofar-deriv This doesn't seem right, We don't have `contract_store`, The `ContractStore` exists in the `contract_replay` and `contract_trade` 🤔 

<img width="477" alt="image" src="https://user-images.githubusercontent.com/72082844/232398945-4a6132e0-971f-42fc-8fc4-30005b78df4a.png">